lib/galaxy/datatypes/binary.py

```diff
@@ -126,6 +126,35 @@ def sniff(self, filename):
             return False
 
 
+class Gpr(Binary):
```

Comment:
By looking at the test file and the online description of the file format, it seems this can be a subclass of `Tabular` or at least `Text`, or am I missing something?